gapic/src/main/com/google/gapid/perfetto/Unit.java

```diff
@@ -195,13 +195,18 @@ public String format(double value) {
     }
 
     public static class Percent extends Simple {
+      private static final Double CUT_OFF = 0.05;
```

Comment:
This should not be a boxed double. Also, add a blank line between the constant and the constructor.